Dockerfile

```
@@ -0,0 +1,16 @@
+FROM node:12-alpine AS base
+RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app
+WORKDIR /home/node/app
+COPY package.json .
+
+FROM base AS build
+RUN apk add --no-cache git bash autoconf automake libtool binutils gcc g++ make python
+USER node
+WORKDIR /home/node/app
+RUN npm install --no-optional --production
```

If we want this to use a package-lock.json file, this should be `npm 

```suggestion
RUN npm ci --no-optional --production
```